DependencyInjection/EkinoNewRelicExtension.php

```diff
@@ -103,6 +118,9 @@ public function load(array $configs, ContainerBuilder $container): void
             $loader->load('twig.xml');
         }
 
+        if (!$config['enabled']) {
+            $config['monolog']['enabled'] = false;
+        }
         if ($config['monolog']['enabled']) {
```

Comment:
`if (!$config['enabled'] && $config['monolog']['enabled'])` ... and we could remove the `if` above, right?

Reply:
done